src/main/drivers/stm32/usb_msc_f7xx.c

```diff
@@ -1,110 +0,0 @@
-/*
```

Comment:
Seems to be [wrongly] used by G4

Reply:
No, I renamed the file to `hal` and missed this one in the make files